packages/imperative/src/profiles/src/abstract/AbstractProfileManager.ts

```diff
@@ -200,12 +200,12 @@ export abstract class AbstractProfileManager<T extends IProfileTypeConfiguration
         if (parms.loadCounter != null) {
             this.mLoadCounter = parms.loadCounter;
         }
-        this.mLogger = isNullOrUndefined(parms.logger) ? this.mLogger : parms.logger;
+        this.mLogger = parms.logger == null ? this.mLogger : parms.logger;
```

Comment:
It is possible that this comment may apply to other files...

Curious if we should take advantage of other operators like `??` 😋

Reply:
I was just doing a find and replace. I was not evaluating any of the other logic. I would think if we want to change something like this, it might make more sense as another issue in order to reduce scope creep.